test/runScript-mock-findBin.spec.js

```diff
@@ -30,9 +28,8 @@ const packageJSON = {
 }
 
 describe('runScript with absolute paths', () => {
-  beforeEach(() => {
-    mockFn.mockReset()
-    mockFn.mockImplementation(() => Promise.resolve(true))
+  afterEach(() => {
+    mockFn.mockClear()
```

Comment:
Do we need to do this for every test? Why?

Reply:
> Resets all information stored in the mockFn.mock.calls and mockFn.mock.instances arrays.
Often this is useful when you want to clean up a mock's usage data between two assertions.

Docs - http://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockclear

Since we check the calls for the mock, we should reset that after each test.

Reply:
Thanks!